src/lib/loaders/api/index.js

```diff
@@ -25,6 +26,7 @@ export default requestIDs => ({
    */
   diffusionLoaderWithoutAuthenticationFactory: apiLoaderWithoutAuthenticationFactory(diffusion, "diffusion", {
     requestIDs,
+    requestThrottleMs: config.DIFFUSION_REQUEST_THROTTLE_MS,
```

Comment:
If I understand correctly, this new 10 minute expiration will apply to all requests made to Diffusion (i.e., for artists' auction results lists, individual auction result views, and the market insights component's requests)? And it will be applied per-metaphysics process and per-unique set of parameters (e.g., artist id)?

Reply:
> If I understand correctly, this new 10 minute expiration will apply to all requests made to Diffusion (i.e., for artists' auction results lists, individual auction result views, and the market insights component's requests)? 

Yes, by configuring this at this level, (and with an `ENV` value that's specific to Diffussion generally), that would apply to all Diffusion loaders created with this factory. Currently, we just have https://github.com/artsy/metaphysics/blob/362efca3fa9e0ceed7da386b95724b7b7a8a35b3/src/lib/loaders/loaders_without_authentication/diffusion.js#L9  (which serves artists' auction results and market insights, which just grabs the first result for a given artist with a special sort).

> And it will be applied per-metaphysics process and per-unique set of parameters (e.g., artist id)?

Yep, the throttler/refresh is in-memory, and doesn't use the cache (so its per-process). And yep the key for the throttler is the same as the underlying cache key, so it includes params.

Reply:
I didn't realize this wasn't yet used for the other auction result views.

This makes sense. Would be good to be able to tailor this by context rather than across all of Diffusion when the time comes, because market insights could afford longer expiration than those.

Reply:
> I didn't realize this wasn't yet used for the other auction result views

Yea, https://www.artsy.net/artist/andy-warhol/auction-results for example, still uses Backbone/Gravity (which just proxies to Diffusion now).